app/serializers/api/v3/finance/statement_serializer.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require "jsonapi/serializer/instrumentation"
+
+module Api
+  module V3
+    module Finance
+      class StatementSerializer
+        include JSONAPI::Serializer
+        include JSONAPI::Serializer::Instrumentation
+
+        set_id :id
+        set_type :statement
+
+        attribute :month do |statement|
+          statement.name[/([^\s]+)/]
```

Comment:
I've used the name to get this value, as the payment date can change manually in the db